src/Modules/NetworkUnsupportedModal.tsx

```diff
@@ -21,10 +21,10 @@ const useStyles = makeStyles(({ constants, palette }: ITheme) =>
     inner: {
       width: "100% !important",
       maxWidth: "unset !important",
-      borderRadius: "4px",
+      borderRadius: 5,
       display: "flex",
       flexDirection: "row",
-      padding: `${constants.generalUnit * 6}px ${constants.generalUnit * 7}px`,
+      padding: `${constants.generalUnit * 3}px ${constants.generalUnit * 3}px`,
```

Comment:
Remove `px`

Reply:
Need to mention the measuring units for shorthand properties.